tests/calculations/test_q2r.py

```diff
@@ -44,11 +44,11 @@ def test_q2r_default(fixture_sandbox, generate_calc_job, generate_inputs_q2r, fi
 @pytest.mark.parametrize(
     ('key', 'value'),
     [
-        ('la2F', True),
+        ('la2f', True),
         ('zasr', 'simple'),
     ],
 )
-def test_q2r_parameters(fixture_sandbox, generate_calc_job, generate_inputs_q2r, file_regression, key, value):
+def test_parameters(fixture_sandbox, generate_calc_job, generate_inputs_q2r, file_regression, key, value):
```

Comment:
This change is to avoid issues with updating the data regression file for `la2f`. `pytest`'s data regression _is_ case sensitive when it looks for the YAML file to compare with, but `git` is not always case sensitive. So by renaming the test and regenerating the data regression files we avoid nasty issues where git doesn't properly track the changes.